### PR TITLE
Add content

### PR DESCRIPTION
**Pull request by jburel**
* Add actions
* Review readme
* add changelog and manifest

### CODE REVIEW

**Comment by will-moore** [comment]
I ran cookiecutter from this branch with:

```
cookiecutter https://github.com/jburel/cookiecutter-omero-webapp.git --checkout actions
```

and pushed to https://github.com/will-moore/omero-cookiecutter-actions-test

There's some formatting that looks wrong for the README badge
```
[![Actions Status]({ cookiecutter.github_repository_url }}/workflows/OMERO/badge.svg)]({ cookiecutter.github_repository_url }}/actions)
```
and the action failed at https://github.com/will-moore/omero-cookiecutter-actions-test/actions/runs/9226643538/job/25386867687

**Comment by jburel** [comment]
thanks. i will have a look

**Comment by will-moore** [comment]
That fixed the badge - pushed it to https://github.com/will-moore/omero-cookiecutter-actions-test

**Comment by jburel** [comment]
@will-moore the build problem should now be fixed.

**Comment by jburel** [comment]
@will-moore #4 is also fixed

**Comment by will-moore** [comment]
Pushed app with those latest changes to https://github.com/will-moore/omero-cookiecutter-actions-test2
There's some formatting issue with the README.

And build is getting further but failing with:
`ModuleNotFoundError: No module named 'omero_cookiecutter_actions_test2'`

**Comment by will-moore** [comment]
That `endraw/raw` change fixed #4 👍 
Not sure I quite understand why `endraw` comes before `raw`!?

**Comment by jburel** [comment]
> That `endraw/raw` change fixed #4 👍 Not sure I quite understand why `endraw` comes before `raw`!?

cookiecutter used jinja2 to replace values in brackets, in the template you do not want to change ``{{ experimenterId }}`` (it will fail anyway) the usage to ``%raw`` allows to handle situation with brackets

there is a ``raw`` earlier in the file

**Comment by jburel** [comment]
The module is called ``cookiecutter_actions_test`` i.e. the omero is dropped 
Test infra assume that the name of the app is for example ``omero-figure`` then we have ``omero_figure``
This is currently not the case in the setup

**Comment by jburel** [comment]
@will-moore I have made a few improvements. The name and repo needs to match in a way for test-infra 
i..e 
```
"webapp_name": "omero_simple_webapp",
"repo_name": "omero-simple-webapp",
```
Build should this time be fixed

**Comment by will-moore** [comment]
Latest test pushed to https://github.com/will-moore/omero-actions-test

```
(omeroweb2024) Williams-MacBook-Pro:OME2024 wmoore$ cookiecutter https://github.com/jburel/cookiecutter-omero-webapp.git --checkout actions
You've downloaded /Users/wmoore/.cookiecutters/cookiecutter-omero-webapp before. Is it okay to delete and re-download it? 
[y/n] (y): y
  [1/11] full_name (OME Team): Will Moore
  [2/11] email (ome-team@openmicroscopy.org): w.moore@dundee.ac.uk
  [3/11] webapp_name (omero_simple_webapp): omero_actions_test
  [4/11] webapp_title (Simple Webapp): OMERO Actions Test
  [5/11] description (OMERO example minimal web app): Testing JM's PR
  [6/11] repo_name (omero-simple-webapp): omero-actions-test
  [7/11] github_username_or_organization (ome): will-moore
  [8/11] github_repository_url (https://github.com/will-moore/omero-actions-test): 
  [9/11] version (0.1.0.dev0): 
  [10/11] development_status (2 - Pre-Alpha): 
  [11/11] copyright_holder (University of Dundee): 
```

**Comment by jburel** [comment]
The failure reported by @will-moore is due to the fact that the generated repository does not have any test included 
see https://github.com/pytest-dev/pytest/issues/662 for background

**Comment by will-moore** [comment]
Made a few commits at https://github.com/will-moore/omero-actions-test and it's now green.

File: {{cookiecutter.repo_name}}/test/integration/test_index.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        group = self.new_group(perms='rwra--')
+        user = self.new_client_and_user(group=group)
+        return user
+
+    def test_load_index(self, user1):
+        """Test loading the app home page."""
+        conn = get_connection(user1)
+        user_name = conn.getUser().getName()
+        django_client = self.new_django_client(user_name, user_name)
+        list_files_url = reverse('omero_actions_test_index')
```
Comment (will-moore) [inline {{cookiecutter.repo_name}}/test/integration/test_index.py]: This url name needs to be same as in urls.py: `"{{cookiecutter.webapp_name}}_index"`.
Test failing at new repo https://github.com/will-moore/omero-actions-pr

Comment (jburel) [reply {{cookiecutter.repo_name}}/test/integration/test_index.py]: I see

**Review by will-moore — APPROVED** [review]
Great - working now. 
Initial commit pushed to https://github.com/will-moore/omero-test-build gets a green build. 👍

**Pull request merged**